database/seeders/ApiEndpointsSeeder.php

```diff
@@ -2398,6 +2398,74 @@ private function seedSummitEndpoints()
                     IGroup::SummitAdministrators,
                 ]
             ],
+            // extra questions
+            [
+                'name' => 'get-sponsor-extra-questions',
+                'route' => '/api/v1/summits/{id}/sponsors/{sponsor_id}/extra-questions',
```

Comment:
we need to create metadata endpoint check SDS

GET /api/v1/summits/{id}/sponsors/all/extra-questions/metadata